mreg/api/v1/tests/test_permissions.py

```diff
@@ -63,6 +63,20 @@ def get_mock_request(user: mock.Mock, mock_user_from_request: mock.Mock, path: s
     return request
 
 
+def create_network_and_permissions(
+    *groups: str,
```

Comment:
This syntax breaks my brain. I get it, but.. It's very magical and weird to read, but that might just be me?

Reply:
The alternative is always passing in a list of groups no matter what. I don't _love_ `*args`, but this is one of the few cases where it's acceptable in my mind.

Reply:
I just prefer the explicit type (I would be fine if we the syntax had been something like `*groups: list[str]` and have single arguments be coerced into a list, but seeing just `str` really hurts my head. Anyway, it's just me being silly.